### PR TITLE
ProgressMonitor

### PR DESCRIPTION
**Pull request by SPalominos**
Execute the wps processes of the server inside SwingWorkerPM to link a progress monitor to the WPS script execution. The ProgressMonitor is given to the groovy script under the variable named 'progressMonitor'. (issue #1169 and #1156)

### CODE REVIEW

**Comment by nicolas-f** [comment]
As a side note. Just take care not calling System.currentTimeMillis() too often. I don't known the calling rate of the method getStatus(GetStatus getStatus)

**Comment by SPalominos** [comment]
The method getStatus(GetStatus getStatus) isn't called often, around one time each 10 minutes.

**Comment by ebocher** [comment]
@SPalominos you mean 10 s.
@nicolas-f  do you have any alternative ?

**Comment by nicolas-f** [comment]
I post a comment yesterday but it fall into a "review" new feature of github. I don't understand how it works.

**Comment by nicolas-f** [comment]
@ebocher if it is each 10s or even each 100 ms it is ok.

File: bundles/wps/wps-client/src/main/java/org/orbisgis/wpsclient/impl/WpsClientImpl.java [hunk 1]
```diff
@@ -175,8 +175,8 @@ public void activate(){
         lee = new LogEditableElement();
         openEditorList = new ArrayList<>();
         le = new LogEditor(lee);
-        dockingManager.addDockingPanel(le);
-        openEditorList.add(le);
+        //dockingManager.addDockingPanel(le);
+        //openEditorList.add(le);
```
Comment (SPalominos) [inline bundles/wps/wps-client/src/main/java/org/orbisgis/wpsclient/impl/WpsClientImpl.java]: My bad, I fix it.

**Pull request merged**